mpf/plugins/twitch/twitch_client.py

```diff
@@ -152,13 +82,15 @@ def on_all_events(self, c, e):
     def do_command(self, e, cmd):
         """Handle a chat command (starts with ? or !)."""
         user = e.source.split('!')[0]
-        self.log.info('Received command: [' + user + '] ' + cmd)
+        self.log.info('Received command: [' + user + '] ' + cmd.lower())
```

Comment:
This would ideally be `self.log.info('Received command: [%s] %s', user, cmd.lower())`. This will safe CPU cycles when logs are not generated as string processing is rather expensive